src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -863,10 +876,10 @@ def checkm2(root_dir, kbase_collection, load_ver, fatal_ids):
     return checkm2_docs
 
 
-def _retrieve_sample(root_dir, kbase_collection, load_ver):
-    print(f'Parsing sample data for {kbase_collection} collection, version {load_ver}.')
-
-    source_dir = os.path.join(root_dir, loader_common_names.COLLECTION_SOURCE_DIR, kbase_collection, load_ver)
+def _retrieve_sample(root_dir, env, kbase_collection, source_ver, load_ver):
+    print(f'Parsing sample data for {kbase_collection} collection, load version {load_ver}, '
+          f'source version {source_ver}.')
+    source_dir = form_source_dir(root_dir, env, kbase_collection, source_ver)
```

Comment:
Hmm, sample data is kind of odd because there's nothing in `collectionsdata`, unlike all the other data, right? I'm wondering if we should have softlinks or something rather than having this one off. I think that's the only reason this script needs to know the `source_ver` as well, right?

On the other hand, this might be a lot easier to deal with once we swap all these files for a database, since we could record the load_ver -> source_ver relationship when it's first created.

Reply:
we don't specify load version during download process. Where are we establishing the softlinks?

Reply:
> we don't specify load version during download process.

Good point. I guess it's either an oddball here or an oddball there

> Where are we establishing the softlinks?

I was thinking of something along the lines of linking `collectionsdata/<env>/<coll>/<load_ver>` to `collectionssource/<env>/<coll>/<source_ver>`. Moot anyway given the above.

Reply:
Although - samples' load_ver will always be the same as the source_ver since there's no calculation step. Anyway, probably easier to deal with in the database based implementation